File: JS/771-jewels-and-stones.js
/**
 * @param {string} J
 * @param {string} S
 * @return {number}
 */
var numJewelsInStones = function(J, S) {
  let count = 0;
  let jewels = J.split("");
  let stones = S.split("");
  
  jewels.forEach(jewel => {
  stones.forEach(stone => {
    if (jewel === stone) {
      count ++
    }
  })
});
  return count;
};

File: PY/771-jewels-and-stones.py
 class Solution(object):
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        count = 0
        for jewel in J: # same syntax for "for...in" loops whether string or array
          for stone in S:
            if jewel == stone:
                count += 1
        return count
